### ISSUE TITLE
MARO-BTC Ticker

### ISSUE THREAD

**Issue by sviriden** [Bittrex/bittrex.github.io#198]
pair **MARO-BTC**

Endpoint: https://api.bittrex.com/v3/markets/MARO-BTC/ticker

Results: 
```
{
 "symbol":"TTC-BTC",
 "lastTradeRate":"0.00000172",
 "bidRate":"0.00000172",
 "askRate":"0.00000192"
}
```

Why MARO changed to TTC ?

**Comment by bittrex-pete** [comment]
Hi @sviriden TTC was renamed to MARO at the request of the team that runs that blockchain. It's interesting that you saw an inconsistent state in the API responses. That issue seems to have resolved itself but it's something we'll investigate. Thanks for reporting it.

**Comment by bittrex-pete** [comment]
For how long were you able to reproduce this issue?

**Comment by sviriden** [comment]
According to my logs, 15.10.2020, UTC time:

15:04 BTC-MARO were added to bittrex (I was using v3 "GET markets/summaries" endpoint to get this info)
15:20 I've started to receive exceptions from my code because of broken results with "MARO-BTC"

For how long — I don't remember, I've fixed my code and cleaned logs with those errors.

But later I also saw similar situations, with incorrect "MarketSummary" results.
I will write you if will see same behaviour again.
